### PR TITLE
[TM-ONLY]Customisation

### PR DESCRIPTION
**Pull request by Vizoplez**
Skyrat Customisation

### CODE REVIEW

File: code/modules/mob/new_player/mutant_accessory/wings.dm [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+/datum/mutant_accessory/wings/wide/megamoth
+	name = "Megamoth"
+	icon_state = "megamoth"
+	dimension_x = 96
+	center = TRUE
+	dimension_y = 32
+	color_src = USE_ONE_COLOR
+	default_color = "FFFFFF"
+
```
Comment (Swiftfeather) [inline code/modules/mob/new_player/mutant_accessory/wings.dm]:
```suggestion
```
Please these. They're too big and don't fold in. Will send you .dmi with the sprites removed as well.

Comment (NotRanged) [reply code/modules/mob/new_player/mutant_accessory/wings.dm]: They scan stay in the .dmi with just the options removed tbh

Comment (Vizoplez) [reply code/modules/mob/new_player/mutant_accessory/wings.dm]: In reality this has had zero effect on gameplay for a variety of reasons such as bump slashing.

Comment (Swiftfeather) [reply code/modules/mob/new_player/mutant_accessory/wings.dm]: > In reality this has had zero effect on gameplay for a variety of reasons such as bump slashing.

I can't pick shit up or click machines when big wings moth man is in the way. It's stupid

**Comment by InterroLouis** [comment]
You massacred my boy. How horrible.

**Comment by Smudged** [comment]
Why remove the option for large wings?

**Comment by InterroLouis** [comment]
Maybe they just didn't want to put the effort in to make them closable.

**Comment by Smudged** [comment]
If this PR does remove mega moth wings, it's kinda just not a good change. It only really impacts the user of Mega moth wings, and not really anybody else, it also puts the marine themself in a disadvantage.

If anything it'd be better to just add in the ability to close and open the wings.

**Comment by Vizoplez** [comment]
> If this PR does remove mega moth wings, it's kinda just not a good change. It only really impacts the user of Mega moth wings, and not really anybody else, it also puts the marine themself in a disadvantage.
> 
> If anything it'd be better to just add in the ability to close and open the wings.

We'll add a way to open-close wings sometime.

**Pull request closed without merging**